backend/src/main/java/gov/cdc/ReportVision/handlers/ValidationExceptionHandler.java

```diff
@@ -0,0 +1,31 @@
+package gov.cdc.ReportVision.handlers;
+
+
+import jakarta.validation.ConstraintViolationException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@ControllerAdvice
+public class ValidationExceptionHandler {
+    @ExceptionHandler(ConstraintViolationException.class)
+    public ResponseEntity<?> handlerException(ConstraintViolationException e) {
+        final List<Object> errors = new ArrayList<>();
+        e.getConstraintViolations().stream().forEach(fieldError -> {
+            Map<String, Object> error = new HashMap<>();
+            error.put("path", String.valueOf(fieldError.getPropertyPath()));
+            error.put("messgae", fieldError.getMessage());
+            errors.add(error);
+        });
+        HttpStatus httpStatus = HttpStatus.BAD_REQUEST;
+        Map<String, Object> body = new HashMap<>();
```

Comment:
`Map<String, Object> body = Map.of("error", errors);`

Since the map would be immutable.

Reply:
Good catch